.prettier.config.js

```diff
@@ -1,35 +1,35 @@
-'use strict';
+"use strict";
```

Comment:
Codacy has a fix for the issue: [Replace `"use·strict"` with `'use·strict'`](https://app.codacy.com/gh/openinf/openinf.github.io/pullRequest?prid=9929905)
```suggestion
'use strict';
```